tests/components/channel_identifier_router/actions.test.js

```diff
@@ -53,4 +54,40 @@ describe('Actions', () => {
             expect(emitChannelClickEvent).toHaveBeenCalledWith(channel2);
         });
     });
+
+    describe('goToDirectChannelByUserId', () => {
+        test('switch to a direct channel by user id on the same team', async () => {
+            const testStore = await mockStore(initialState);
```

Comment:
Do mockStore need to be async?

Reply:
I'm not entirely sure, I was just copying the usages I saw elsewhere